vendor/modules.txt

```diff
@@ -12,6 +12,7 @@ github.com/Azure/go-autorest/logger
 # github.com/Azure/go-autorest/tracing v0.5.0
 github.com/Azure/go-autorest/tracing
 # github.com/aws/aws-sdk-go v1.25.48
+## explicit
```

Comment:
Did you run `go mod tidy` which added this?

Reply:
@sebsoto please add the commands you ran to make the change to the commit message.